README.md

```diff
@@ -52,7 +52,47 @@ Generate team
 
 #### Configuring GatorGrade Checks
 
-Input team
+There are multiple customizable options with gatorgrade!
+Gatorgrader checks are able to run within a specific file context
+and without any file path.
```

Comment:
To make sure the content of `README.md` remains consistent, could you capitalize "GatorGrade" and "GatorGrader" like 👈  😃 ?